java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +371,43 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Pops one or more elements from the first non-empty list <code>key</code> from the list of
+     * provided <code>key</code> names
```

Comment:
```suggestion
     * Pops one or more elements from the first non-empty list from provided <code>keys</code>.
```
Please reword. ^ this is an example